src/main/java/org/example/camunda/process/solution/service/OperateService.java

```diff
@@ -10,12 +10,19 @@
 import io.camunda.operate.search.Sort;
 import io.camunda.operate.search.SortOrder;
 import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.cache.annotation.EnableCaching;
 import org.springframework.stereotype.Service;
 
 @Service
+@EnableCaching
```

Comment:
I'm not sure caching is a must in such case. It's useful to manage eviction (with TTL for example) but the embedded forms are tightly coupled with the process definition and not subject to change. What would it bring ?

Reply:
Also do we want caching on OperateService or BpmnUtils ?